src/hooks/useFetch.js

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from "react";
```

Comment:
Perhaps we could pass in URL and query as params instead of hardcoding so we can reuse this hook elsewhere?

Reply:
Now passing URL as a parameter to the hook .